src/control/mod.rs

```diff
@@ -1450,3 +1463,14 @@ bitflags::bitflags! {
         const ATOMIC = ffi::DRM_MODE_PROP_ATOMIC;
     }
 }
+
+bitflags::bitflags! {
+    /// Planar framebuffer flags
+    #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+    pub struct FbCmd2Flags : u32 {
+        /// For interlaced framebuffers
+        const INTERLACED = ffi::DRM_MODE_FB_INTERLACED;
+        /// Enables .modifier
```

Comment:
Nit: would be great if this was an intradoc-link to ``[`PlanarInfo::modifier()`]`` and/or `PlanarBuffer::modifier()`?